<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="simpleGroupBox">
     <property name="title">
      <string>Settings:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QToolButton" name="dbButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="dbEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="dbLabel">
        <property name="text">
         <string>Database file: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="thresholdLabel">
        <property name="text">
         <string>Default duplicates threshold: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="thresholdSlider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="thresholdSpinBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="advancedGroupBox">
     <property name="title">
      <string>Advanced Settings:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLineEdit" name="pdftexEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="languagetoolLabel">
        <property name="text">
         <string>Languagetool:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="pdftexButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="pdftexLabel">
        <property name="text">
         <string>pdflatex:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="laguagetoolEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="languagetoolLanguage">
        <item>
         <property name="text">
          <string>ast-ES</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>be-BY</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>br-FR</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ca-ES</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ca-ES-valencia</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>zh-CN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>da-DK</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>nl</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>en</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>en-AU</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>en-CA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>en-GB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>en-NZ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>en-US</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>eo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>fr</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gl-es</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>de</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>de-AT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>de-DE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>de-CH</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>el-GR</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>it</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ja-JP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>km-KH</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>fa</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>pl-PL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>pt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>pt-AO</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>pt-BR</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>pt-MZ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>pt-PT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ro-RO</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ru-RU</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>sr</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>sr-BA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>sr-HR</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>sr-ME</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>sr-RS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>de-DE-x-simple-language</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>sk-SK</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>sl-SI</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>es</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>sv</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>tl-PH</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ta-IN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>uk-UA</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
